src/commands/cliOptions.js

```diff
@@ -29,10 +29,13 @@ module.exports = {
     origin: ['--origin <remotename>', 'the name of the remote repo', 'origin'],
     releaseBranch: ['--release-branch <branch>', 'the target branch for the release PR', 'master'],
     wwwUser: ['--www-user <user>', 'the user who runs php commands', 'www-data'],
+    updateTranslations: ['--update-translations', 'indicates if we need to update translations. Only available in interactive mode.', true],
```

Comment:
Still not sure about the way this option is expressed. I ran an interactive `taoRelease extensionRelease --update-translations=false`, and was still prompted for the translation step, which I though was strange.

Then on a hunch I re-checked the [commander docs](https://github.com/tj/commander.js#default-option-value), and specifying a default value for a flag isn't a supported syntax. The flag is either present, or not, but never takes a value. Their convention is to add `--flag` for setting a param true, and `--no-flag` for setting it false.

In fact, I realise that the meaning of the old `--update-translations` was "don't prompt me this time, just do it". If we stick to that meaning and stay closer to the old code, we can continue to have that in interactive mode, and still turn it off in non-interactive mode, WDYT?

Reply:
I've restored the original behavior of the flag which is more about the prompt itself.